pkg/api/validation/validation_test.go

```diff
@@ -3864,6 +3864,14 @@ func getResourceList(cpu, memory string) api.ResourceList {
 	return res
 }
 
+func getNonStandardContainerResourceList(storage string) api.ResourceList {
```

Comment:
call this `getStorageResourceList(storage string) api.ResourceList` for clarity.